.travis.sh

```diff
@@ -1,34 +1,38 @@
 #!/bin/bash
 
 # common tasks before either build or test
-function prepare ()
+function configure()
 {
-  set -e   # Make sure any error makes the script to return an error code
-  set -x   # echo
-
-  SOURCE_DIR=`pwd`
-  BUILD_DIR=build
-
-  #env
-  git clean -fd || true
-  rm -fr $BUILD_DIR || true
-  mkdir $BUILD_DIR && cd $BUILD_DIR
-
-  if [ -z "$CMAKE_BUILD_TYPE" ]; then
-    CMAKE_BUILD_TYPE=Debug
-  fi
-
-  if [ -z "$GTSAM_ALLOW_DEPRECATED_SINCE_V4" ]; then
-    GTSAM_ALLOW_DEPRECATED_SINCE_V4=OFF
-  fi
-
-  if [ ! -z "$GCC_VERSION" ]; then
-    sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-$GCC_VERSION 60 \
-                         --slave /usr/bin/g++ g++ /usr/bin/g++-$GCC_VERSION
-    sudo update-alternatives --set gcc /usr/bin/gcc-$GCC_VERSION
-  fi
+	set -e   # Make sure any error makes the script to return an error code
```

Comment:
indentation

Reply:
Oops! I've been playing these days switching forth and back between new IDEs, it seems in some of them I forgot to enable "show invisibles" Thanks! :+1: